log/_state.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+
+import { BaseHandler } from "./base_handler.ts";
+import { DEFAULT_CONFIG, DEFAULT_LEVEL } from "./_config.ts";
+import { type LogConfig, Logger } from "./logger.ts";
+
+export const state = {
+  handlers: new Map<string, BaseHandler>(),
+  loggers: new Map<string, Logger>(),
+  config: DEFAULT_CONFIG,
+};
```

Comment:
Can you please move this to a separate `_state.ts` file and rename this current file `setup.ts`?

Reply:
Updated.